submissions/nhunzaker/lib/server.js

```diff
@@ -0,0 +1,28 @@
+var Hapi = require('hapi')
+var pkg = require('../package')
+
+exports.metadata = {
```

Comment:
Although `exports.metadata` is simply to expose the data `/version` sends over. It isn't a Hapi thing